docs/csharp/language-reference/operators/subtraction-operator.md

```diff
@@ -1,35 +1,74 @@
 ---
-title: "- operator - C# Reference"
+title: "- and -= operators - C# Reference"
 ms.custom: seodec18
-
-ms.date: 07/20/2015
+ms.date: 05/27/2019
 f1_keywords: 
   - "-_CSharpKeyword"
+  - "-=_CSharpKeyword"
 helpviewer_keywords: 
+  - "subtraction operator [C#]"
+  - "delegate removal [C#]"
   - "- operator [C#]"
-  - "subtraction operator (-) [C#]"
+  - "subtraction assignment operator [C#]"
+  - "event unsubscription [C#]"
+  - "-= operator [C#]"
 ms.assetid: 4de7a4fa-c69d-48e6-aff1-3130af970b2d
 ---
-# - operator (C# Reference)
+# - and -= operators (C# Reference)
+
+The `-` operator is supported by the built-in numeric types and [delegate](../keywords/delegate.md) types.
+
+For information about the arithmetic `-` operator, see the [Unary plus and minus operators](arithmetic-operators.md#unary-plus-and-minus-operators) and [Subtraction operator -](arithmetic-operators.md#subtraction-operator--) sections of the [Arithmetic operators](arithmetic-operators.md) article.
+
+## Delegate removal
+
+For operands of the same [delegate](../keywords/delegate.md) type, the `-` operator returns a delegate instance that is calculated as follows:
+
+- If both operands are non-null and the invocation list of the second operand is a proper contiguous sublist of the invocation list of the first operand, the result of the operation is a new invocation list that is obtained by removing the second operand's entries from the invocation list of the first operand. If the second operand's list matches multiple contiguous sublists in the first operand's list, only the right-most matching sublist is removed. If removal results in an empty list, the result is `null`.
+- If the invocation list of the second operand is not a proper contiguous sublist of the invocation list of the first operand, the result of the operation is the first operand.
```

Comment:
Related to my comments on dotnet/samples#929  This is the condition that often surprises developers. An example would help.